packages/@okta/vuepress-site/docs/guides/manage-orgs-okta-aerial/main/index.md

```diff
@@ -0,0 +1,378 @@
+---
+title: Manage orgs with Okta Aerial
+excerpt: Learn how to create and manage orgs added to your company's Okta Aerial account.
+layout: Guides
+sections:
+  - main
+---
+
+This guide explains how to add orgs to your Okta Aerial account, and manage Aerial org settings.
+
+> **Note:** Access to Okta Aerial requires an additional product to be present. Ensure that your Okta account team gives you access to the Okta Aerial API.
+
+---
+
+**Learning outcomes**
+
+- Authenticate an Aerial API client with Okta Aerial.
+- Add an org to your Aerial account.
+- Enable products in the Aerial linked org.
+- Configure the Aerial org.
+
+**What you need**
+
+- An Okta Aerial account
+- A parent org
+- Access to the Org creator API
+- Access to the Okta Aerial API
+
+**Sample code**
+
+Use the [Okta Aerial API](link to redocly)
+
+---
+
+## About Okta Aerial
+
+Okta Aerial is an administration service that enables multi-org management. Use the Aerial API to:
+
+- Add orgs to the Aerial account.
+- Activate and deactivate orgs.
+- View subscribed products.
+- Enable products in specific orgs.
+
+### Key terms
+
+Okta Aerial introduces a few terms to the Okta ecosystem:
+
+<dl>
+<dt><strong>Aerial account</strong></dt>
+<dd>The management layer around multiple orgs within Okta. The Aerial account lives outside of your orgs and can manage any production or preview org linked to the Aerial account.
+</dd>
+</br>
+<dt><strong>Aerial org</strong></dt>
+<dd>Holds the authorization server for all Aerial API actions in any org in the Aerial account. Choose one org to permanently serve as the Aerial Org. <br/>Super admins can create API clients in the Aerial Org to access the Aerial account. The Aerial Org also contains all System Log events associated with Okta Aerial actions.</dd>
+</br>
+<dt><strong>Products</strong></dt>
+<dd>Okta-determined sets of features. Example products include: `P000052`: IT Products - SSO, `P000131`: IT Products - MFA, `P000139`: IT Products - Lifecycle Management with 1 OIN App. With the Okta Aerial API, you can view subscribed products for an Aerial account and enable a subset of products to orgs.</dd>
+</br>
+<dt><strong>Features</strong></dt>
+<dd>Distinct pieces of functionality. Features are bundled within products but may also be offered separately, for example, Early Access features.</dd>
+</dl>
+
+### Aerial sandbox and production environments
+
+Okta provides the following environments to use with your Aerial account:
+
+- Sandbox: Only connected to OP3 cells. Use for scale testing and repetitive testing.
+
+  `https://aerial-sandbox/{accountId}`
```

Comment:
`https://aerial-sandbox.okta.com/{accountId}`

We can also probably drop `/{accountId}`.